spec/chromium-spec.js

```diff
@@ -955,19 +955,41 @@ describe('chromium feature', () => {
       slashes: true
     })
 
-    function createBrowserWindow ({plugins}) {
+    function createBrowserWindow ({plugins, preload}) {
       w = new BrowserWindow({
         show: false,
         webPreferences: {
-          preload: path.join(fixtures, 'module', 'preload-pdf-loaded.js'),
+          preload: path.join(fixtures, 'module', preload),
           plugins: plugins
         }
       })
     }
 
-    it('opens when loading a pdf resource as top level navigation', (done) => {
-      createBrowserWindow({plugins: true})
-      ipcMain.once('pdf-loaded', (event, state) => {
+    function testPDFIsLoadedInSubFrame (page, preloadFile, done) {
+      const pagePath = url.format({
+        pathname: path.join(fixtures, 'pages', page).replace(/\\/g, '/'),
+        protocol: 'file',
+        slashes: true
+      })
+
+      createBrowserWindow({plugins: true, preload: preloadFile})
+      ipcMain.once('pdf-loaded', function (event, state) {
```

Comment:
Is it possible that `'pdf-loaded'` listener will be called before the one of `'page-title-updated'`?

Reply:
Its possible if the title is updated in javascript. But it shouldn't happen for this spec.